ifrs17-template/Report/ReportEyCases.ipynb

```diff
@@ -47,7 +47,17 @@
         {
             "cell_type": "code",
             "source": [
-                "#!import \"../Initialization/InitSystemorphToMemory\""
+                "#!import \"../Initialization/InitSystemorphUtils\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await InitializeDataNodesAsync(DataSource, pathToImport, new string []{\"FR\"});",
+                "\nawait InitializeTransactionalDataAsync(DataSource, pathToImport, new string[] {\"FR\"});"
```

Comment:
Having one method Initializing all the data (DataNode and Transactional) for the company is nicer.

Reply:
Done.